### ISSUE TITLE
Value of type 'JSON' has no member 'collection'

### ISSUE THREAD

**Issue by theabhishek2511** [freshOS/Arrow#18]
Hi, I'm trying to iterate a JSON like so

```
if let collection = json.collection {
        for jsonEntry in collection {
            // things
        }
    }
```

and it says there is no member 'collection' in json. Checking the source code here shows that there actually is a 'collection'. I tried readding the libraries through Carthage as well but no dice.

**Comment by MontakOleg** [comment]
@theabhishek2511 this is because `collection` has **internal** access modifier by default (https://github.com/s4cha/Arrow/blob/master/Source/JSON.swift#L24)

I think it should be public.

**Comment by s4cha** [comment]
@MontakOleg Nice catch! For some obscure reasons I could access it on my end :/
@theabhishek2511  This should be fixed now in latest version (2.0.2)

Cheers,
